### PR TITLE
Story/ccp 117

### PR DESCRIPTION
**Pull request by mkarthik90**
[CCP-117] As per comments removed implements POS. Just importing the package is enough,

### CODE REVIEW

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
I can't store a concept wrapper after modifying it. According to the Jira story, this should be fixed now?

**Comment by mkarthik90** [comment]
@jdamerow  For editing concept wrapper. I tested editing the name of concept List as per story CCP-110 it was working fine. Also I tried to edit concepts within concept list. I am able to edit successfully. I think the issue should be with the old corrupted data. Please recreate the database and try out once. 

Also please let me know if you are talking about a different issue.

**Comment by jdamerow** [comment]
retest this please

**Comment by jdamerow** [comment]
retest this please

**Comment by mkarthik90** [comment]
@jdamerow  When i created a new user in dev region and edited the concept entries in concept list, its working fine.

**Comment by mkarthik90** [comment]
@jdamerow  I have resolved the merge conflicts. Could you please verify and merge the branch

**Comment by jdamerow** [comment]
ok to test

**Comment by jdamerow** [comment]
I still can't store a modified concept wrapper.

**Comment by mkarthik90** [comment]
@jdamerow  Did you try with username other than admin. I think it will work for users other than admin which we discussed couple of days ago

**Comment by mkarthik90** [comment]
@jdamerow  Made the changes for adding the admin username. Since the admin user name and details are not stored in the db while fetching the values a Null pointer exception is thrown. So made the changes accordingly and verified store modified concept.

**Comment by jdamerow** [comment]
retest this please

File: Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
 	@PostConstruct
 	public void init() {
 		admins = new HashMap<String, String>();
+		admins.put("admin","admin");
```
Comment (jdamerow) [inline Conceptpower+Spring/src/main/java/edu/asu/conceptpower/users/impl/UsersManager.java]: no, don't hard-code username and password! Add the ones from the sprint-security file.

**Pull request closed without merging**